docs/conf.py

```
@@ -127,7 +127,7 @@
 # The short X.Y version.
 version = "v2.0"
 # The full version, including alpha/beta/rc tags.
-release = "v2.0.9a6"
+release = "v2.0.9a7"
```

Pylint (3.3.7)

[convention] Constant name "release" doesn't conform to UPPER_CASE naming style

(C0103)